samples/MvvmSampleUwp/MvvmSampleUwp/Assets/docs/PuttingThingsTogether.md

```diff
@@ -155,9 +155,10 @@ We now have a draft of our viewmodels ready, and we can start looking into the s
 
 ## Building the settings service
 
-> **NOTE:** the sample is built using the service locator pattern, but this is not the only possible pattern to use to manage service. The MVVM Toolkit also fully supports the dependency injection pattern, and you can choose the one you prefer depending on the architecture of your application, the available development time or personal preference.
+> [!NOTE]
+> The sample is built using the dependency injection pattern, which is the recommended approach to deal with services in viewmodels. It is also possible to use other patterns, such as the service locator pattern, but the MVVM Toolkit does not offer built-in APIs to enable that.
```

Comment:
Do we want to be careful here with saying 'the recommended' approach? Maybe 'a commonly used'?
```suggestion
> The sample is built using the dependency injection pattern, which is a commonly used approach to deal with services in viewmodels. It is also possible to use other patterns, such as the service locator pattern, but the MVVM Toolkit does not offer built-in APIs to enable that.
```

The service locator requires a platform specific implementation, eh? Do we want to call that out at the end to be clear on _why_ it's not here?

Reply:
It doesn't require a platform specific implementation (eg. `Microsoft.Extensions.DependencyInjection` targets .NET Standard), we just decided not to include one mostly to avoid reinventing the wheel and competing with much larger and specifically purposed libraries anyway. I think I'm calling that out a few lines below this bit? 🤔